sdk/apis/apis/v1alpha2/types_apiexport.go

```diff
@@ -148,7 +149,12 @@ type APIExportSpec struct {
 }
 
 type ResourceSchema struct {
-	// Schema is the name of the referenced APIResourceSchema.
+	// Name is the name of the resource.
+	Name string `json:"name"`
```

Comment:
this seems off, compare https://github.com/kcp-dev/kcp/pull/3318/commits/d5e6f866419346c71e16397929091b87108d8d8c#r2006280842.

Would rename the field to `spec.resource`. Then you naming makes sense.

Reply:
Renamed to resources, but object stays `ResourceSchema` due to fact we already have func named `Resource`